tests/integration/API/Catalog/Product_Item/ResponseTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace SkyVerge\WooCommerce\Facebook\Tests\API\Catalog\Product_Item;
+
+use SkyVerge\WooCommerce\Facebook\API\Catalog\Product_Item\Response;
+
+/**
+ * Tests the Connection class.
```

Comment:
Needs to be updated.

Reply:
I finally enabled spell checking in Visual Studio Code for the typos 😅, now I need something for this kind of copy🍝

Thank you for catching that one! Fixed in cef61cd.